pages/common/fswatch.md

```diff
@@ -0,0 +1,19 @@
+# fswatch
+
+> A cross-platform file change monitor.
+
+- Run a bash command on file creation, update or deletion:
+
+`fswatch -0 {{path/to/watch}} | xargs -0 {{bash_command}}`
+
+- Watch (multiple) directories or files
```

Comment:
1) why is "multiple" in parenthesis? I don't think that adds anything to the description. 2) I think it's more natural to say "files or directories" than the reverse, but it's ok if you disagree.

Also, you need a : at the end of the description :)